assets/src/setup/components/navigation-context-provider.js

```diff
@@ -24,19 +23,16 @@ export const Navigation = createContext();
  * @param {Array} props.pages Pages in the app.
  */
 export function NavigationContextProvider( { children, pages } ) {
-	// Initialize page from URL `amp-setup-screen` parameter. If not set, current page is 0.
-	// This is primarily for testing.
-	const [ activePageIndex, setActivePageIndex ] = useState( () => {
-		const index = pages.findIndex( ( { slug } ) => slug === getQueryArg( global.location.href, 'amp-setup-screen' ) );
```

Comment:
Aside: I miss this query var 😄  It was useful to be on a screen and reload after changes to avoid having to tap Next Next Next.

Reply:
Should I put it back? I thought I was probably the only person using it.

Reply:
Yes, please do. It's nice for development, no?